src/local-time.lisp

```diff
@@ -1008,7 +1008,10 @@ elements."
     (let ((err (ccl:external-call "gettimeofday" :address tv :address (ccl:%null-ptr) :int)))
       (assert (zerop err) nil "gettimeofday failed")
       (values (ccl:pref tv :timeval.tv_sec) (* 1000 (ccl:pref tv :timeval.tv_usec)))))
-  #-(or allegro cmu sbcl (and ccl (not windows)))
+  #+abcl
+  (let ((millis (java:jstatic "currentTimeMillis" "java.lang.System")))
+    (values (truncate millis 1000) (* (mod millis 1000) 1000000)))
```

Comment:
`truncate` returns the remainder as the second value. Please use `multiple-value-bind` to avoid dividing twice.